cards.js

```diff
@@ -17,7 +17,63 @@ function shuffle(array){
 }
 
 function buildDeck(){
+  var deck=[];
+  for(var i = 0; i < values.length; i++){
+    for(var j = 0; j < suits.length; j++){
+      var temp_hash ={};
+      temp_hash = {value: values[i], suit: suits[j], score: i+2};
+      deck.push(temp_hash);
+    };
+  };
 
 // Put some code in here!
+return deck;
+}
+
+var warDeck = buildDeck();
+var shuffledDeck = shuffle(warDeck);
+// console.log(warDeck);
 
+function highCard(deck) {
+  var playerOne = deck.pop();
+  var playerTwo = deck.pop();
+  console.log(playerOne);
+  console.log(playerTwo);
+  if(playerOne.score > playerTwo.score){
```

Comment:
Nice job on implementing the `.score` attribute. It took me a minute to understand how it worked, but this is clever!